src/ducks/Signals/SignalMaster.js

```diff
@@ -85,8 +99,11 @@ export class SignalMaster extends React.PureComponent {
     } else {
       this.props.createTrigger({ ...this.state.trigger, ...about })
     }
-    this.props.onCreated && this.props.onCreated()
-    this.props.redirect()
+    if (this.props.onCreated) this.props.onCreated()
```

Comment:
![image](https://user-images.githubusercontent.com/522287/59060340-54d43a00-88ba-11e9-820a-7006f961c7b5.png)

Why not the same design of {} you used.
if () {
}

Reply:
Fixed